tyrant/__main__.py

```diff
@@ -10,23 +10,28 @@
 
 # Initialize the bot
 bot = bot.Tyrant(
-    command_prefix=when_mentioned_or(constants.Bot.prefix),  # Invoked commands must have this prefix
+    command_prefix=when_mentioned_or(
+        constants.Bot.prefix
+    ),  # Invoked commands must have this prefix
```

Comment:
```suggestion
    command_prefix=when_mentioned_or(constants.Bot.prefix),
```
I don't like this, feels like it has worse readability. Let's put it back in one line, but we can remove the superfluous comment to bring the line length down. This comment isn't very helpful.